css/main.css

```diff
@@ -3,13 +3,12 @@ html {
 }
 
 body {
-	 min-height: 100%;
-     padding: 30px;
-     padding-top: 70px;
-     font-family: 'Roboto', sans-serif;
-     display: flex;
-	 flex-direction: column;
-	 font-size-adjust: inherit;
+	min-height: 100%;
+    padding-top: 70px;
+    font-family: 'Roboto', sans-serif;
+    display: flex;
+	flex-direction: column;
+	font-size-adjust: inherit;
```

Comment:
nit: Spacing is a bit weird here.

Reply:
doesn't look like that on my editor. i'll repush.